portal-api/src/v1/gef/controllers/application.controller.js

```diff
@@ -283,25 +287,25 @@ exports.findDeals = async (requestingUser, filters, start = 0, pagesize = 0) =>
   const collection = await db.getCollection(dealsCollection);
 
   const doc = await collection
-    .aggregate([ // TODO SR-8
-      { $match: sanitisedFilters },
+    .aggregate([
+      { $match: { $eq: sanitisedFilters } },
       {
         $sort: {
-          updatedAt: -1,
-          createdAt: -1,
+          updatedAt: -1, // TODO SR-8: Sort order -- no $eq expression needed
+          createdAt: -1, // TODO SR-8: Sort order -- no $eq expression needed
         },
       },
       {
         $facet: {
-          count: [{ $count: 'total' }],
-          deals: [{ $skip: start }, ...(pagesize ? [{ $limit: pagesize }] : [])],
+          count: [{ $count: 'total' }], // TODO SR-8: Does not take an expression -- no $eq expression needed
+          deals: [{ $skip: start }, ...(pagesize ? [{ $limit: pagesize }] : [])], // TODO SR-8: Does not take an expression -- no $eq expression needed
```

Comment:
Remove todo once noted:
```suggestion
          count: [{ $count: 'total' }],
          deals: [{ $skip: start }, ...(pagesize ? [{ $limit: pagesize }] : [])],
```